src/lua_resty_netacea.lua

```diff
@@ -352,19 +508,39 @@ function _N:ingest()
     NetaceaMitigationApplied = ngx.ctx.bc_type
   }
 
-  local command = table.concat({
-    "curl",
-    "-X POST",
-    "-m 5",
-    "-s",
-    "-o /dev/null",
-  "-H 'Content-Type:application/json'",
-    "-H 'x-netacea-api-key:" .. self.apiKey .. "'",
-    "-d '" .. cjson.encode(data) .. "'",
-    self.ingestEndpoint
-  }, " ")
-
-  os.execute(command .. " &")
+  -- start STASH code
+  local request_params = {};
+
+  request_params.body  = cjson.encode(data);
+  request_params.method  = "POST";
+  request_params.headers = {
+        ["Content-Length"] = #request_params.body,
+        ["Content-Type"] = "application/json",
+        ["x-netacea-api-key"] = self.apiKey;
+  }
+  request_params.timeout = 1000; -- 1 second
+
+  local request_task = function()
+        local hc = http:new();
```

Comment:
There's some very funky indentation throughout, could you run the file through something to fix this?

Reply:
Original file was all 2 space, this seems to have some 4 and some 6 in places?